### ISSUE TITLE
Не работает с выходом новой версии jellyfin

### ISSUE THREAD

**Issue by iAHTOH** [LinFor/jellyfin-plugin-kinopoisk#9]
После добавления репозитория отсутсвует в каталоге Jellyfin 10.8.8 
https://github.com/jellyfin/jellyfin/releases/tag/v10.8.8

**Comment by MrJosu** [comment]
да, тоже самое :( 
К сожалению больше сопоставление не работает

**Comment by skrashevich** [comment]
Собрал версию работающую на 10.8.8. Пока автор здесь не обновил, можно обновить из моего репозитория: https://raw.githubusercontent.com/skrashevich/jellyfin-plugin-kinopoisk/master/dist/manifest.json

**Comment by melkosoft** [comment]
@skrashevich  Ваша версия для 10.8.9 - не работает...По крайней мере с моим api key

**Comment by skrashevich** [comment]
@melkosoft  Запускаете в докере? Какая ОС? Что в логе при запуске? (там где инициализируются все остальные плагины)

**Comment by skrashevich** [comment]
Try to disable TMDB plugin. The are banned Russian IP's on their API, so plugin generate this error.
TMDB plugin loads before KinoPoisk (but after IMDB) -- and affects on it

**Comment by melkosoft** [comment]
Those plugins are built-in - no button to disable or uninstall like manually installed

**Comment by skrashevich** [comment]
> Those plugins are built-in - no button to disable or uninstall like manually installed

You can disable it in the media library properties

**Comment by melkosoft** [comment]
it was my mistake - i didn't enable plugin in media library properties. everything is working except posters but may be i missed some settings somewhere...

**Comment by skrashevich** [comment]
> 
Check the "Image fetchers" Library settings. Move this plugin above any others

**Comment by melkosoft** [comment]
done. thank you for your help

**Comment by vsenn** [comment]
> Собрал версию работающую на 10.8.8. Пока автор здесь не обновил, можно обновить из моего репозитория: https://raw.githubusercontent.com/skrashevich/jellyfin-plugin-kinopoisk/master/dist/manifest.json

@skrashevich Спасибо за поддержку плагина Кинопоиск в своей репе. К сожалению, последняя версия 10.8.9.3 у меня не работает, выдавая ошибку про SSL сертификат. Возможно где-то в коде её необходимо отключить.

```
Provider КиноПоиск failed to retrieve search results
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure: RemoteCertificateNameMismatch, RemoteCertificateChainErrors
```

**Comment by skrashevich** [comment]
@vsenn А у вас с того же устройства, где запущен jellyfin, в браузере https://kinopoiskapiunofficial.tech нормально открывается?

**Comment by vsenn** [comment]
Да, с этим сайтом всё в порядке, открывается по https без ошибок 
![image](https://github.com/LinFor/jellyfin-plugin-kinopoisk/assets/25433230/1b5e0f9e-2b6d-45c3-8c35-6cb355a45b73)
